worlds/generic/docs/commands_en.md

```diff
@@ -1,96 +1,124 @@
+- `!hint_location <location>` Tells you what item is in a specific location, uses points earned from completing locations.
+
+### Collect/Release
+- `!collect` Grants you all the remaining items for your world by collecting them from all games. Typically used after 
+goal completion.
+- `!release` Releases all items contained in your world to other worlds. Typically, done automatically by the sever, but
+can be configured to allow/require manual usage of this command.
+
+### Cheats
+- `!getitem <item>` Cheats an item, if it is enabled in the server.
+
+### Broken/Deprecated/WIP
+- `!status` Returns information about your team. <br /> (Currently all players as teams are unimplemented. Instead,
+this shows the connection status and check completion numbers for all players in the current room.)
+
+
+## Host only (on Archipelago.gg or in your server console)
+
+### General
+- `/help` Returns a list of commands available in the console.
+- `/license` Returns the software licensing information.
+- `/options` Lists the server's current options, including password in plaintext.
+- `/players` List currently connected players.
+- `/save` Saves the state of the current multi-world. Note that the server auto-saves on a minute basis.
+- `/exit` Shutdown the server
+
+### Utilities
+- `/countdown <number of seconds>` Starts a countdown sent to all players via text chat. Defaults to 10 seconds if no
+  argument is provided.
+- `/option <option name> <option value>` Set a server option. For a list of options, use the `/options` command.
+- `/alias <player name> <alias name>` Assign a player an alias, allowing you to reference the player by the alias in commands.
+
+
+### Collect/Release
+- `/collect <player name>` Send out any items remaining in the multi-world belonging to the given player.
+- `/release <player name>` Sends out all remaining items in this world regardless of settings and game completion status.
+- `/allow_release <player name>` Allows the given player to use the `!release` command.
+- `/forbid_release <player name>` Prevents the given player from using the `!release` command.
+
+### Cheats
+- `/send <player name> <item name>` Grants the given player the specified item.
+- `/send_multiple <amount> <player name> <item name>` Grants the given player the stated amount of the specified item.
+- `/send_location <player name> <location name>` Send out the given location for the specified player as if the player checked it
+- `/hint <player name> <item or location name>` Send out a hint for the given item or location for the specified player.
+
+<br/> <br/>
+
+# Local Commands
+
+This a list of client commands which may be available to you through your Archipelago client. You can
 execute these commands in your client window.
 
-The following commands are available in these clients: SNIClient, FactorioClient, FF1Client.
+The following commands are available in these clients: TextClient, SNIClient, FactorioClient, FF1Client.
```

Comment:
If I understand, all of the clients that get installed into the Archipelago folder support these commands, right?
Perhaps a good way to differentiate these from other games such as Stardew Valley, Muse Dash, or Dark Souls III would be an emphasis on the external client outside of the game.

Please correct me if I am wrong about any of these assumptions.

Reply:
Looking through all the clients that get added to the folder, I see two games that seem to not use `ClientCommandProcessor` which from my understanding is for the commands. Those games being Minecraft and LADX.

Reply:
>Those games being Minecraft and LADX.

I did some brief research into these two clients to see how they handled Archipelago local commands.

LADX appears to work the same as other clients on the end user side. The normal set of local commands are available and work normally.

Minecraft does NOT allow these local commands, and instead when using ```/help```, it defaults to the Minecraft command of the same name. I believe this to be the case because the Minecraft "client" is simply a mode of running a Minecraft Forge server and connecting to AP servers. Therefore, Minecraft does not have local commands like other CommonClients.

Right now, the issue that I am facing is that all of the clients allow these local commands with the exception of Minecraft. Therefore, it's not correct to say that all clients in the Archipelago install support these. However, I believe that mentioning that only Minecraft does not support these commands takes the issue we have now and flips it on its head, just causing issues later down the line when that list inevitably becomes outdated.

The only thread that I can find that ties everything together for the end user is that they all have the style of the TextClient.
![image](https://github.com/ArchipelagoMW/Archipelago/assets/79181893/d2311684-9ba5-4ac3-afbf-6d8c9650220a)

Another thing to note is that you do not technically have to run the Minecraft client yourself in order to launch your world. The ```APMC``` file will do this for you automatically when you attempt to open it through double-click or the like. In contrast, the other clients need to be run separately from their games and patches to function.

One possible way around this issue is renaming ```ArchipelagoMinecraftClient.exe``` to something like ```ArchipelagoMinecraftServerHost.exe``` or something, that way all of the clients can be referred to at once. This fix is imperfect itself, but I think that it is an easier fix than trying to recategorize clients.

Reply:
We could maybe just generalize the way it’s saying it by something like `The following commands are available in the clients that use CommonClient, for example: TextClient, SNIClient etc` There’s probably a better way to say it though.

I don’t like changing the name of Minecraft.

Reply:
That's a solution I can get behind, I will commit a change and then I think this PR is ready unless you have other reservations.